labs/lab3/chapter3.md

```diff
@@ -176,123 +176,102 @@ Save the Dockerfile and exit the editor.
+  * `-p <container_port>` to map the container port to the host port
+
 ```bash
-$ ls -lZd /var/lib/mariadb
-$ docker run -d -v /var/lib/mariadb:/var/lib/mysql:Z -p 3306:3306 -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb --name mariadb mariadb
+$ ls -lZd ~/workspace/wp_logs
+$ sudo podman run -d -p 8080 -v ~/workspace/wp_logs:/var/log/httpd:z -e DB_ENV_DBUSER=user -e DB_ENV_DBPASS=mypassword -e DB_ENV_DBNAME=mydb -e DB_HOST=0.0.0.0 -e DB_PORT=3306 --name wordpress wordpress
 ```
-
-Note: See the difference in SELinux context after running w/ a volume & :Z.
+Note: See the difference in SELinux context after running w/ a volume & :z.
 ```bash
-$ ls -lZd /var/lib/mariadb
-$ docker exec $(docker ps -ql) ps aux
+$ ls -lZd ~/workspace/wp_logs
+$ sudo podman exec wordpress ps aux
 ```
 
 Check volume directory ownership inside the container
 ```bash
-$ docker exec $(docker ps -ql) stat --format="%U" /var/lib/mysql
-$ docker logs $(docker ps -ql)
-$ docker ps
-$ curl localhost:3306
+$ sudo podman exec wordpress stat --format="%U" /var/log/httpd/access_log
+$ sudo podman logs wordpress
+$ sudo podman ps
 ```
 
   **Note**: the `curl` command does not return useful information but demonstrates
             a response on the port.
 
 5. Test the Wordpress image to confirm connectivity. Additional run options:
-  * `--link <name>:<alias>` to link to the database container
+  * `--network=container:<alias>` to link to the wordpress container
 ```bash
-$ ls -lZd /var/lib/wp_uploads
-$ docker run -d -v /var/lib/wp_uploads:/var/www/html/wp-content/uploads:Z -p 8080:8080 --link mariadb:db --name wordpress wordpress
+$ ls -lZd ~/workspace/mysql
+$ sudo podman run -d --network=container:wordpress -v ~/workspace/logs:/var/log/mariadb:z -v ~/workspace/mysql:/var/lib/mysql:z -v ~/workspace/run:/run/mariadb:z -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb --name mariadb mariadb
 ```
-
-Note: See the difference in SELinux context after running w/ a volume & :Z.
+Note: See the difference in SELinux context after running w/ a volume & :z.
 ```bash
-$ ls -lZd /var/lib/wp_uploads
-$ docker exec $(docker ps -ql) ps aux
+$ ls -lZd ~/workspace/mysql
+$ sudo podman exec mariadb ps aux
 ```
 
 Check volume directory ownership inside the container
 ```bash
-$ docker exec $(docker ps -ql) stat --format="%U" /var/www/html/wp-content/uploads
-$ docker logs $(docker ps -ql)
-$ docker ps
+$ sudo podman exec mariadb stat --format="%U" /var/log/mariadb/mariadb.log
+$ sudo podman logs mariadb
+$ sudo podman ps
+$ sudo podman exec mariadb curl localhost:3306
 $ curl -L http://localhost:8080
 ```
 
 You may also load the Wordpress application in a browser to test its full functionality @ `http://<YOUR AWS VM PUBLIC DNS NAME HERE>:8080`.
 
 ### Simplify running containers with the atomic CLI
```

Comment:
i think we should possibly nuke this section. atomic CLI doesn't seem to work with podman and buildah [doesn't yet support caching build layers for `buildah bud`](https://github.com/projectatomic/buildah/issues/601) so we highlight the fact that changing one line in the Dockerfile requires it to be rebuilt from scratch.